Makefile

```diff
@@ -15,7 +15,7 @@
 #    limitations under the License.
 #
 
-PYTHON ?= python3
+PYTHON ?= ./env/bin/python3
```

Comment:
When venv is activated, can we access just by `python3`?

Reply:
Yes, only if `activate` is called first and then `deactivate` is called after. I feel this is a simpler change for the makefile.